### ISSUE TITLE
EBE-based diagnostics differ for PK and PD when there are missing PD (or PK)

### ISSUE THREAD

**Issue by baltcir1** [ggPMXdevelopment/ggPMX#205]
Version 1.2.6. Just an observation (it could be considered as a bug) that is a "side effect" of the ggPMX design: when there are more than one endpoints (PKPD examples), we create one controller per endpoint. 

Suppose we have missing PD measurements for some patients. When we crease the PD controller, I guess we filter and keep only the IDs that have observable PD. As a result, all EBE-based diagnostics (which should be the same as for PK controller), such as pmx_plot_eta_matrix, pmx_plot_eta_cats, pmx_plot_eta_conts, will be different in PD report because we have omitted individuals that miss PD. BTW, in Monolix, this doesn't happen.

So, basically, the correct EBE-based diagnostics should be those of the endpoint that has less missing values (in my example, the diagnostics produced with the PK controller). It can be misleading for example when you look at pmx_plot_eta_conts, the correlation coefficient won't be the same for PK and PD controller:

PK controller:
![image](https://user-images.githubusercontent.com/25454764/154454487-e39cb109-59fc-4a1c-a592-8e5d79dac585.png)

PD controller:
![image](https://user-images.githubusercontent.com/25454764/154454554-2857c77e-d890-4277-8fc6-51b27c5afe1f.png)
